tracer/src/Datadog.Trace/Ci/CITracerManagerFactory.cs

```diff
@@ -77,12 +77,14 @@ protected override IAgentWriter GetAgentWriter(ImmutableTracerSettings settings,
         private IApiRequestFactory GetRequestFactory()
         {
             IApiRequestFactory factory = null;
+            TimeSpan agentlessTimeout = TimeSpan.FromSeconds(20);
```

Comment:
20s still seems quite long to me? I was thinking more like 5s, especially with the retries? IMO, a smaller timeout, and larger delay between retries may be preferable, but not passionate about it

Reply:
yes, we decide 15secs, but the Timeout in .NET includes the DNS resolver, and that can take 15secs 🤦🏻 . So I need to go above of that number.
https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.timeout?view=net-6.0#remarks

Reply:
oh wait, is separated, not included. Will reduce it to 15secs.

Reply:
done in [667db3a](https://github.com/DataDog/dd-trace-dotnet/pull/2548/commits/667db3a76d26152885c3807a251572ea737c9c9f)